governance/default.go

```diff
@@ -401,6 +402,21 @@ func (gs *GovernanceSet) Merge(change map[string]interface{}) {
 }
 
 func NewGovernance(chainConfig *params.ChainConfig, dbm database.DBManager) *Governance {
+	return &Governance{
+		ChainConfig:              chainConfig,
+		voteMap:                  NewVoteMap(),
+		db:                       dbm,
+		itemCache:                newGovernanceCache(),
+		currentSet:               NewGovernanceSet(),
+		changeSet:                NewGovernanceSet(),
+		lastGovernanceStateBlock: 0,
+		GovernanceTallies:        NewGovernanceTallies(),
+		GovernanceVotes:          NewGovernanceVotes(),
+		idxCacheLock:             new(sync.RWMutex),
+	}
+}
+
```

Comment:
What is difference between `NewGovernance ` and `NewGovernanceInitialize`?
Could you make comments?

Reply:
I added comments.